src/app/components/SocialMedia/index.tsx

```diff
@@ -1,16 +1,13 @@
 import styled from "styled-components/macro";
-import { ReactComponent as InstaIcon} from '../../../../assets/images/Instagram.svg';
-import { ReactComponent as FacebookIcon } from '../../../../assets/images/Facebook.svg';
-import { ReactComponent as YoutubeIcon} from '../../../../assets/images/YouTube.svg';
+import { ReactComponent as InstaIcon} from '../../../assets/images/Instagram.svg';
+import { ReactComponent as FacebookIcon } from '../../../assets/images/Facebook.svg';
+import { ReactComponent as YoutubeIcon} from '../../../assets/images/YouTube.svg';
+
```

Comment:
This is the diff between some of your versions. As it is the first PR the diff should be between nothing and everything. The reason is the branch where you are merging already contains some commits. You can reset the branch to the very beginning and then force push it. It might require recreation of the PR